Assets/Scripts/Audio/AudioController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/** The master audio manager--loads banks and 
+ contains functions that will start, stop, pause, resume sound events*/
+    
+public class AudioController : MonoBehaviour {
+
+    uint bankID;
+    //used in stop,resume, pause event functions
+    //give value of fadeout to know how much to fade in and out of the sound
+    public int fadeout; 
+    //load banks; currently will hardcode in bankss
+    public void LoadBanks()
+    {
+        AkSoundEngine.LoadBank("Incinerator", AkSoundEngine.AK_DEFAULT_POOL_ID, out bankID);
+    }
+
+    //plays the event 
+    public void PlayEvent( string eventName )
+    {
+        Debug.Log("Currently playing event");
```

Comment:
Please remove or comment out debugging statements. They should only be used while they are actively needed (or we'll end up clogging up the console very quickly).